packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -86,7 +86,7 @@ export class AtomicQuickviewModal implements InitializableComponent {
 
   private interactiveResult?: InteractiveResult;
 
-  public componentWillRender(): void {
+  public componentWillLoad(): void {
```

Comment:
https://stenciljs.com/docs/component-lifecycle#componentwillrender: every time it renders, it conflicts with how the state works.
https://stenciljs.com/docs/component-lifecycle#componentwillload: run once.